### PR TITLE
impersonate user in new tab

### LINKED ISSUE #10886: Customer impersonation bug when no referrer url sent
**Sylius version affected**: 1.6

**Description**  
Impersonation controller in the admin throws unhandled 500 error when no referrer url is available.

**Steps to reproduce**  
1. Go to https://demo.sylius.com/admin/customers/
2. `Show` any customer
3. Copy `Impersonate` button's URL
4. Paste in the browser
5. Kaboom

**Possible Solution**  
Check referrer was sent. If not redirect to the client's "show" page

### PR DESCRIPTION
**Pull request by oallain**
| Q               | A
| --------------- | -----
| Branch?         | 1.6
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes #10886 
| License         | MIT

That solve the issue, but I don't know how to test it with Behat.

### CODE REVIEW

**Comment by oallain** [comment]
Hi @lchrusciel @Zales0123 ,

I add behat tests for this feature.

Regards

File: src/Sylius/Bundle/AdminBundle/Controller/ImpersonateUserController.php [hunk 1]
```diff
@@ -73,7 +75,13 @@ public function impersonateAction(Request $request, string $username): Response
 
         $this->addFlash($request, $username);
 
-        return new RedirectResponse($request->headers->get('referer'));
+        $redirectUrl = $request->headers->get('referer');
```
Comment (Zales0123) [inline src/Sylius/Bundle/AdminBundle/Controller/ImpersonateUserController.php]: I suppose we can do it with the default value as the second parameter:
```php
$redirectUrl = $request->headers->get(
    'referer',
    $this->router->generate('sylius_admin_customer_show', ['id' => $user->getId()])
);
```

Comment (oallain) [reply src/Sylius/Bundle/AdminBundle/Controller/ImpersonateUserController.php]: Thanks :)

**Comment by pamil** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "impersonate-new-tab"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by pamil** [comment]
Thanks, @oallain! :1st_place_medal:

**Pull request merged**